web/docs/migrate-from-0-13-to-0-14.md

```diff
@@ -10,258 +10,437 @@ This guide only covers the migration from **0.13.X to 0.14.X**. If you are migra
+Wasp introduced a much simpler API for accessing user auth fields like `username`, `email` or `isEmailVerified` on the `user` object. You don't need to use helper functions every time you want to access the user's `username` or do extra steps to get proper typing.
 
 ## How to migrate?
 
+### Migrate to the new `schema.prisma` file
+To use the new `schema.prisma` file, you need to move your entities from the `.wasp` file to the `schema.prisma` file.
+
+1. **Create a new `schema.prisma` file**
+
+   Create a new file named `schema.prisma` in the root of your project:
+
+   ```c
+    .
+    ├── main.wasp
+    ...
+    // highlight-next-line
+    ├── schema.prisma
+    ├── src
+    ├── tsconfig.json
+    └── vite.config.ts
+   ```
+2. **Add the `datasource` block** to the `schema.prisma` file
+
+   This block specifies the database type and connection URL:
+
+   ```prisma title="schema.prisma"
+   datasource db {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+   ```
+
+   - The `provider` should be either `"postgresql"` or `"sqlite"`. 
+   
+   - The `url` must be set to `env("DATABASE_URL")` so that Wasp can inject the database URL from the environment variables.
+
+3. **Add the `generator` block** to the `schema.prisma` file
+
+   This block specifies the Prisma Client generator Wasp uses:
+
+   ```prisma title="schema.prisma"
+   datasource db {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+
+   // highlight-start
+   generator client {
+     provider = "prisma-client-js"
+   }
+    // highlight-end
+   ```
+
+   - The `provider` should be set to `"prisma-client-js"`.
+
+4. **Move your entities** to the `schema.prisma` file
+
+   Move the entities from the `.wasp` file to the `schema.prisma` file:
+
+   ```prisma title="schema.prisma"
+   datasource db {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+
+   generator client {
+     provider = "prisma-client-js"
+   }
+
+   // There are some example entities, you should move your entities here
+   // highlight-start
+   model User {
+     id       Int @id @default(autoincrement())
+     tasks    Task[]
+   }
+
+   model Task {
+     id          Int @id @default(autoincrement())
+     description String
+     isDone      Boolean
+     userId      Int
+     user        User @relation(fields: [userId], references: [id])
+   }
+   // highlight-end
+   ```
+
+   When moving the entities over, you'll need to change `entity` to `model` and remove the `=psl` and `psl=` tags.
+
+   One trick you could use to make the migration faster is to copy the entities from the `.wasp/out/db/schema.prisma` file to the new `schema.prisma` file. But, be careful not to copy over the internal Wasp entities like `Auth`, `AuthIdentity` and `Session`. You should only copy the entities you defined.
```

Comment:
I might even be more in favor of keeping this to ourselves.

The users who know what's going on will know how to do it themselves and won't need us to tell them. The users who don't know what's going on are arguably better off not digging around the generated files, as they'll very likely copy something they shouldn't have (despite this warning).

Then again, it does make migration much easier. Hmm, you call.

Reply:
Yeah. you are probably right. 

We inject a relationship to the `User` model as well, so they'd need to remove it which some people might not realise. This tip is for internal use only then!